From d61db320d8a893a292812256470a873c7ce1b35d Mon Sep 17 00:00:00 2001
From: Eric Koegel <eric.koegel@gmail.com>
Date: Mon, 15 Sep 2014 14:35:00 +0300
Subject: [PATCH] Fix handle lid switch with loginD (Bug 11160)

The xfconf settings logic for logind interactions were inverted
in commit: 03193df6cad651841e8d9960335b4a3bf13afdad
However the handle lid switch in the xfpm_manager_lid_changed_cb
also needs to be inverted. This patch fixes it. Patch submitted
by Simon P.
---
 src/xfpm-manager.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/xfpm-manager.c b/src/xfpm-manager.c
index d6c533c..9549b66 100644
--- a/src/xfpm-manager.c
+++ b/src/xfpm-manager.c
@@ -335,7 +335,7 @@ xfpm_manager_lid_changed_cb (XfpmPower *power, gboolean lid_is_closed, XfpmManag
               LOGIND_HANDLE_LID_SWITCH, &logind_handle_lid_switch,
               NULL);
 
-        if (!logind_handle_lid_switch)
+        if (logind_handle_lid_switch)
             return;
     }
 
-- 
2.1.0

